src/tools/clippy/clippy_lints/src/missing_doc.rs

```diff
@@ -94,7 +79,7 @@ impl MissingDoc {
         }
 
         let has_doc = attrs.iter().any(|a| {
-            a.is_doc_comment() || a.doc_str().is_some() || a.value_str().is_some() || Self::has_include(a.meta())
+            a.is_doc_comment() || a.doc_str().is_some() || a.value_str().is_some()
         });
```

Comment:
FYI @rust-lang/clippy you should probably be using `rustc_lint::builtin::has_doc` instead; this has a false positive and also counts things like `#[path = "x"]` as doc-comments (because you only check value_str().is_some(), not the name of the attribute). Also in general it seems nice to have all the logic in one place.

Reply:
Looks like a nice cleanup opportunity on the Clippy side! `has_doc` would just have to be made public

Reply:
Sure thing, you could do both at the same time in a PR to rust-lang/rust :)

Reply:
I opened an issue: rust-lang/rust-clippy#7247